DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -224,6 +226,12 @@ public override void Initialize()
             lblCurrentChannel.FontIndex = 1;
             lblCurrentChannel.Text = "CURRENT CHANNEL:";
 
+            lblOnline = new XNALabel(WindowManager);
+            lblOnline.Name = "lblOnline";
+            lblOnline.ClientRectangle = new Rectangle(310, 14, 0, 0);
+            lblOnline.FontIndex = 1;
+            lblOnline.Text = "";
```

Comment:
Call lblOnline.Disable() here to make it disabled by default. Then you can enable it for YR in CnCNetLobby.ini.